third-party/README

```diff
@@ -117,7 +117,7 @@ qthread/
 
 re2/
   Summary: The RE2 regular expression library used to support regular
-           expression operations described in doc/technotes/regexp.rst
+           expression operations described in doc/rst/technotes/regexp.rst
```

Comment:
I can't find a regexp.rst or anything that looks like a regexp technote.

At least now it refers to a directory that exists...

Reply:
it should refer to the Regexp module docs (these used to be in a technote).